content/en/blog/2023/lambda-release.md

```diff
@@ -0,0 +1,54 @@
+---
+title: OpenTelemetry Updates our Lambda Support
+linkTitle: OTel Lambda Updates
+date: 2023-05-15
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+---
+
+The Functions-as-a-Service (FAAS) SIG is incredibly excited to announce that the
+OpenTelemetry (OTel) community is now releasing our own Lambda layers and has
+[centralized documentation](https://opentelemetry.io/docs/faas/) on how to
+monitor Amazon Web Services (AWS) Lambdas.
+
+### A Slightly New World
```

Comment:
The heading level should be at level 2.

```suggestion
## A Slightly New World
```

I'm not sure how much meaning that title text conveys. How about:

```suggestion
## OTel FaaS repackaged
```